cleancode/src/main/java/com/epam/cleancode/comments/InvalidInputException.java

```diff
@@ -1,11 +1,12 @@
 package com.epam.cleancode.comments;
 
-public class InvalidInputException extends RuntimeException{
+public class InvalidInputException extends RuntimeException {
     String message;
+
     public InvalidInputException(String s) {
```

Comment:
Please rename 's' to something more readable

Reply:
` public InvalidInputException(String messageText) {`
   
fix TashaB1@e2848c1